docs/mesos/Troubleshooting.md

```diff
@@ -0,0 +1,32 @@
+<!--- master only -->
+> ![warning](../images/warning.png) This document applies to the HEAD of the calico-containers source tree.
+>
+> View the calico-containers documentation for the latest release [here](https://github.com/projectcalico/calico-containers/blob/v0.18.0/README.md).
+<!--- else
+> You are viewing the calico-containers documentation for release **release**.
+<!--- end of master only -->
+
+# Troubleshooting Calico with Mesos
```

Comment:
Can you yank this file out of this PR? the calico-mesos-framework has gathered a bit of dust, we shouldn't recommend using it atm. I'll add a personal backlog item to scroll through the mesos slack and identify some common issues that I can fill this guide with